src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -222,10 +243,19 @@ private void BuildGlobbedLinkTags(IDictionary<string, string> attributes, TagHel
         private void BuildFallbackBlock(TagHelperContent builder)
         {
             EnsureGlobbingUrlBuilder();
-            var fallbackHrefs = GlobbingUrlBuilder.BuildUrlList(FallbackHref, FallbackHrefInclude, FallbackHrefExclude);
+            var fallbackHrefs =
+                GlobbingUrlBuilder.BuildUrlList(FallbackHref, FallbackHrefInclude, FallbackHrefExclude).ToArray();
 
             if (fallbackHrefs.Any())
             {
+                if (FileVersion == "true")
+                {
+                    for (var i=0; i < fallbackHrefs.Count(); i++)
```

Comment:
`fallbackHrefs.Length` as you made it an array. Calling `.Count()` on each iteration should be a red flag.